tubular/nominal.py

```diff
@@ -537,11 +546,20 @@ class MeanResponseTransformer(BaseNominalTransformer, BaseMappingTransformMixin)
         Only created in the mutli-level case. A dictionary of the form level : transformer containing the mean response
         transformers for each level to be encoded against.
 
+    impute_dict: dict
+        Dict containing the values (based on chosen unseen_levels) derived from the encoded columns to use when handling unseen levels in data passed to transform method.
+
 
     """
 
     def __init__(
-        self, columns=None, weights_column=None, prior=0, level=None, **kwargs
+        self,
+        columns=None,
+        weights_column=None,
+        prior=0,
+        level=None,
+        unseen_levels=None,
```

Comment:
`unseen_level_handling` would be more pythonic and make some later code more readable

Reply:
Yes! unseen_level_handling is much better! Changed unseen_levels to unseen_level_handling in commit: [3307aa84](https://github.com/lvgig/tubular/pull/80/commits/3307aa84031d6356f7e42f62d4bb1e41f70f949d)